tests/test_self.py

```diff
@@ -1293,6 +1293,15 @@ def test_output_no_header(self) -> None:
             args, expected_output=expected, unexpected_output=not_expected
         )
 
+    def test_no_name_in_module(self) -> None:
+        """Test that a package with both a variable name `errors` and a module `errors`
```

Comment:
```suggestion
        """Test that a package with both a variable name `base` and a module `base`
```
Took me some time to understand 😄 I think the errors.py could be outright removed ?

Reply:
Woops you're right that the docstring is wrong! But while errors.py can technically be removed and still get the msg, it makes less sense to do so since the thing that triggers this is exactly this import `from pkg_mod_imports.base.errors import SomeError`

Trying to mimic the original issue raised here.